ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -2628,19 +2629,23 @@ and perform_exn ?result (op : operation) (t : Xenops_task.task_handle) : unit =
           ~path:(Uri.path_unencoded url ^ snippet ^ id_str)
           ~query:(Uri.query url) ()
       in
-      (* CA-78365: set the memory dynamic range to a single value to stop
-         ballooning. *)
-      let atomic =
-        VM_set_memory_dynamic_range
-          (id, vm.Vm.memory_dynamic_min, vm.Vm.memory_dynamic_min)
-      in
-      let (_ : unit) =
-        perform_atomic ~progress_callback:(fun _ -> ()) atomic t
-      in
-      (* Waiting here is not essential but adds a degree of safety and
-         reducess unnecessary memory copying. *)
-      ( try B.VM.wait_ballooning t vm
-        with Xenopsd_error Ballooning_timeout_before_migration -> ()
+      (* CA-78365: set the memory dynamic range to a single value to stop ballooning.
```

Comment:
I think we still need this part: even if you want to avoid ballooning *down*, you still need to *stop* ballooning. A VM attempting to balloon during migration won't be very healthy.

Instead we probably need to look at how much memory it is currently using, and set the balloon target to that to stop it from changing it.

Reply:
Also we should probably skip all this when static_min == dynamic_min == dynamic_max == static_max, which is the usual setting (I'd hope that the code here is mostly a noop in that case, but I'm not sure).

Reply:
Both of these should be done now